### ISSUE TITLE
Improve documentation for measureLayout and findNodeHandle

### ISSUE THREAD

**Issue by Sharcoux** [facebook/react-native-website#2029]
## Description

I am trying to use measureLayout but the doc seems quite complicated to understand.

So, the first argument of `measureLayout` should be a `number`. Doesn't start well but ok.

The doc says:

> measureLayout(relativeToNativeNode, onSuccess, onFail) Like measure(),
> but measures the view relative to an ancestor, specified as
> relativeToNativeNode. This means that the returned x, y are relative
> to the origin x, y of the ancestor view.
> 
> As always, to obtain a native node handle for a component, you can use
> findNodeHandle(component).
> 
> import { findNodeHandle } from 'react-native';

I don't know where this *as always* comes from because it is actually the only reference to this method within the docs of both React and React-Native: [![enter image description here][1]][1]
[![enter image description here][2]][2]

  [1]: https://i.stack.imgur.com/AM56s.png
  [2]: https://i.stack.imgur.com/uL04M.png

So, ok, let's use `findNodeHandle`. It takes as parameter a... *number* oO? or a ReactComponent, or a ReactClass... Well, if I had such a thing at my disposal, I wouldn't need this, would I? What I have is a ref that I created with createRef. So that's what I expect the documentation to show me how to use.

Basically, what I'm trying to show is that from a user's point of view, the current API is nor friendly, nor well documented and could probably be improved.

## Documentation version

0.62

**Comment by Sharcoux** [comment]
Latest documentation is WAAAAAY better. Nice job to the people that made it happen :+1:
